src/components/search/facets/custom-sliced-filters.vue

```diff
@@ -1,13 +1,17 @@
 <template>
   <SlicedFilters v-bind="$attrs">
```

Comment:
I think we should add a dynamic prop to add classes to the `sliced-filters` buttons.
After that, we would just apply the `x-facet-filter-lg` class here.